### PR TITLE
:sparkles: Data responses: GET /applet/{id}/data

### PR DESCRIPTION
**Pull request by shnizzedy**
This new route allows a logged-in user to download all responses for any applet for which they have "reviewer" permissions.

Resolves https://github.com/ChildMindInstitute/MindLogger-bug-reports/issues/51

* The data can be downloaded as a CSV or a JSON file. For some reason, the file names are being prepended and appended with underscores.
* The data format for the CSVs is not yet expanded at all, so the only headers right now are 

  respondant | activity | applet | responses | subject | responseStarted | responseCompleted | client | languageCode | last7Days
  -- | -- | -- | -- | -- | -- | -- | -- | -- | --

  with Objects in most of the cells.
* More tests should be added to catch future security weaknesses.
* This cascade of functions will eventually need to be optimized to scale.

Includes PRs #248 & #253

f7570e01efdb1080d8c704ccb163b3d1a0329f47 is live on dev.

### CODE REVIEW

File: girderformindlogger/utility/jsonld_expander.py [hunk 1]
```diff
@@ -311,7 +315,7 @@ def delanguageTag(obj):
 
 
 def expandOneLevel(obj):
-    if obj==None:
+    if obj is None:
```
Comment (henryrossiter) [inline girderformindlogger/utility/jsonld_expander.py]: Have you considered changing this to ```if not obj:``` ? I don't know all of the use cases here, but ```not obj``` will also catch empty objects whereas ```obj is None``` only catches Nonetype objects. Just curious.

Comment (shnizzedy) [reply girderformindlogger/utility/jsonld_expander.py]: https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/256/commits/65d75e4e905fd910e00b54532c70c8f6d7200fab

**Comment by shnizzedy** [comment]
:warning: 65d75e4 caused excessive recursion, so I reverted and now the tests pass again. This PR is re-ready for review.

File: girderformindlogger/utility/jsonld_expander.py (hunk 1 above)
Comment (shnizzedy) [reply girderformindlogger/utility/jsonld_expander.py]: https://github.com/ChildMindInstitute/mindlogger-app-backend/pull/256#issuecomment-571295712

**Pull request merged**